lib/metasploit/framework/data_service/proxy/nmap_data_proxy.rb

```diff
@@ -0,0 +1,12 @@
+module NmapDataProxy
+
+  def import_nmap_xml_file(args = {})
+    begin
+      data_service = self.get_data_service()
+      data_service.import_nmap_xml_file(args)
+    rescue Exception => e
+      puts "Call to #{data_service.class}#import_nmap_xml_file threw exception: #{e.message}"
```

Comment:
I realized I started this trend ... we need to get jiggy with the correct logging format

Reply:
I totally agree. I'll create a ticket to go back through and add real logging to our code.